LICENSE-MIT

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 chriseth
+Copyright (c) 2023,2024 powdr labs and contributors
```

Comment:
Will we remember to update this every year?

Reply:
idk, you suggested the year in the chat

Reply:
I didn't make up the year in the copyright notice, it is already there in the previous version.

In theory it server to count when the copyright expires. In practice it is silly because this is publicly git versioned.

Reply:
I am in favor of removing this line. The only question is if GitHub will still identify the license.

Reply:
we can try and fix otherwise

Reply:
done